packages/fiori/src/themes/Page.css

```diff
@@ -29,25 +30,45 @@
 }
 
 .ui5-page-footer-root {
+    width: 100%;
+    box-sizing: border-box;
     position: absolute;
     bottom: 0;
     left: 0;
     z-index: 2;
     width: 100%;
 }
 
-:host([disable-scrolling]) .ui5-page-content-root {
-    overflow: hidden;
-}
-
 :host([floating-footer]) .ui5-page-footer-root {
-    left: 0.5rem;
-    right: 0.5rem;
-    width: auto;
+    padding: 0 3rem;
     opacity: 1;
     bottom: 0.5rem;
 }
 
+@media (max-width: 599px) {
```

Comment:
Responsive paddings should be implemented on component size.
Otherwise if you put the Page inside a smaller container, for example column in FCL, instead of S size paddings the component will be rendered with the L,XL size paddings. See the Implementation of the TabContainer for reference.